doc.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "A Social platform for the creative at heart",
+    "version": "1.0.0",
+    "title": "Authors Haven",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "email": "vidar.ah2019@gmail.com"
+    },
+    "license": {
+      "name": "GNU General Public License",
+      "url": "https://www.gnu.org/licenses/gpl-3.0.en.html"
+    }
+  },
+  "host": "authors-haven-staging.herokuapp.com",
```

Comment:
URL should be <repo-name>-staging.herokuapp.com for staging and <repo-name>.herokuapp.com for production.